src/executor/sort.rs

```diff
@@ -107,15 +107,6 @@ impl<'a, T: 'static + Debug> Sort<'a, T> {
                         Some(ord) if ord != Ordering::Equal => {
                             return apply_asc(ord);
                         }
-                        None => {
-                            let a = String::from(value_a);
-                            let b = String::from(value_b);
-                            let ord = a.cmp(&b);
-
-                            if ord != Ordering::Equal {
-                                return apply_asc(ord);
-                            }
-                        }
```

Comment:
I was too rushing on these tasks so explanation wasn't enough.

Previous status was problematic, so I simply removed some unnecessary codes.
However, it still needs to be improved.
(Converting into a comparable value first is required, some enum type which implements `Eq`.)

This `None` happens when `Value` enum types between items are different to each other.
That must not exist before but due to `MAP` type, it can appear.

Your point is right, we actually need to return `Error` rather than `String::from` comparison or using `Equal`.
For this issue, I only considered to use `try_sort` like method, but it looks...
if we convert `Value` first into a same type before we call `sort`, then that can be also a clear solution.

Reply:
That's right, I think it would be better to create a separate issue for this and manage it.
The question has been resolved.
I will approve